update_src_lists

```diff
@@ -0,0 +1,2 @@
+yarn arbify --tokenList ./src/SourceLists/whitelist_era_src.json --l2NetworkID 42161 --includeOldDataFields true
```

Comment:
why not include as part of updateAll script?

Reply:
will move it once it's used directly externally instead of using the static file, as you suggested